src/components/RequestError.js

```diff
@@ -85,7 +88,7 @@ class RequestErrorModal extends React.Component {
               </button>
             </div>
             <div className="modal-body">
-              <p>Your request failed to reach the server. What do you want to do?</p>
+              <p>{this.props.requestErrorMessage} What do you want to do?</p>
```

Comment:
Even though "What do you want to do?" appears in both cases, I think we should move it to the `this.props.requestErrorMessage` (and add them to the end of the messages).

Reply:
Makes sense, so in the future we can have messages without this part